core/src/main/java/io/lionweb/lioncore/java/language/Property.java

```diff
@@ -73,7 +73,7 @@ public Property setType(@Nullable DataType type) {
 
   @Override
   public String toString() {
-    return "Property{" + "name=" + getName() + ", " + "type=" + getType() + '}';
+    return super.toString() + "{" + "name=" + getName() + ", " + "type=" + getType() + '}';
```

Comment:
Why don't construct a _qualifiedName_ like for `LanguageEntity` (for symmetry)?

Reply:
Done